packages/cli/README.md

```diff
@@ -244,10 +257,10 @@ USAGE
 
 OPTIONS
   -c, --ceramic=ceramic  Ceramic API URL
-  -k, --key=key    DID Key
+  -k, --key=key          DID Private Key
```

Comment:
also need to confirm this one.  Depends on what it does. If it's just creating local data and not writing anything to ceramic then I imagine --key isn't used?

Reply:
ditto for all these `model` commands